docker/suricata/dist/suricata.yaml

```diff
@@ -275,6 +280,12 @@ outputs:
             # to an IP address is logged.
             extended: no
         - ssh
+        - mqtt:
+            # passwords: yes           # enable output of passwords
```

Comment:
Do you want to enable MQTT?

Reply:
Yes, makes sense since Dionaea has MQTT enabled.